.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 0
+    requirements:
```

Comment:
I'm not sure if you need to specify the requirements file (or if this is a valid key); can you explain the intent here or link to the docs on it?

Reply:
it's not needed but figured extra specification couldn't hurt